deepchecks/utils/distribution/plot.py

```diff
@@ -204,17 +204,15 @@ def feature_distribution_traces(train_column,
 
         train_density = get_density(train_column, xs)
         test_density = get_density(test_column, xs)
-        bars_width = (x_range_to_show[1] - x_range_to_show[0]) / 100
+        # bars_width = (x_range_to_show[1] - x_range_to_show[0]) / 100
```

Comment:
I believe this won't look well here without width

Reply:
not big difference as for me, but you are right it is not related to our problem, will revert


-- with width --

![image](https://user-images.githubusercontent.com/71635444/168093558-7dcfce52-a165-42d0-9d7c-53dff10ee4df.png)

-- without width --

![image](https://user-images.githubusercontent.com/71635444/168093850-dc189913-ba7c-4043-bd4e-3b393d3e3c2b.png)